<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Chat_demo</class>
 <widget class="QDialog" name="Chat_demo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>827</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Chat</string>
  </property>
  <widget class="QLineEdit" name="Input_field">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>450</y>
     <width>651</width>
     <height>141</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="More">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>450</y>
     <width>161</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string>More</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Send">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>450</y>
     <width>181</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string>Send</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>0</y>
     <width>651</width>
     <height>451</height>
    </rect>
   </property>
  </widget>
  <widget class="QListView" name="listView">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>171</width>
     <height>451</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
